packages/utils/src/TestingUtils/JestUtils/TestWrapper.js

```diff
@@ -5,29 +5,31 @@ import { Provider } from 'react-redux';
 import { MemoryRouter, Route, Routes } from 'react-router-dom';
 import { IntlProvider } from 'react-intl';
 
-const TestWrapper = ({ renderOptions = {}, children }) => {
-  const mockStore = configureStore();
+const mockStore = configureStore();
 
-  return (
-    <IntlProvider locale="en">
-      <Provider store={renderOptions?.store || mockStore()}>
-        <MemoryRouter initialEntries={renderOptions?.initialEntries || ['/']}>
-          {renderOptions?.componentPath ? (
-            <Routes>
-              <Route path={renderOptions?.elementPath || '/'}>{children}</Route>
-            </Routes>
-          ) : (
-            children
-          )}
-        </MemoryRouter>
-      </Provider>
-    </IntlProvider>
-  );
-};
+const TestWrapper = (children, routerProps = { initialEntries: ['/'] }, componentPath, store = mockStore()) => (
```

Comment:
I think the props should be wrapped in an object.

Reply:
Ah, yes!

Reply:
Fixed!